airflow-core/src/airflow/models/pool.py

```diff
@@ -198,9 +197,9 @@ def slots_stats(
         )
 
         # calculate queued and running metrics
-        for pool_name, state, count in state_count_by_pool:
+        for pool_name, state, count_raw in state_count_by_pool:
```

Comment:
```suggestion
        for pool_name, state, decimal_count in state_count_by_pool:
```

maybe `decimal_count`, it expresses why we need to do it better